tests/ingest_runs/test_genomic_data_loader.py

```diff
@@ -0,0 +1,396 @@
+    for key in sg.cache["sequencing_experiment_genomic_file"]:
+        row_dict = {
+            "_links.sequencing_experiment": (
+                ast.literal_eval(key)["sequencing_experiment_id"]
+            ),
+            "_links.genomic_file": ast.literal_eval(key)["genomic_file_id"],
+        }
+        rows.append(row_dict)
+    return pd.DataFrame(rows)
+
+
+def map_seq_exp(sg) -> pd.DataFrame:
+    rows = []
+    for key, value in sg.cache["sequencing_experiment"].items():
+        row_dict = {
+            "kf_id": value,
+            "external_id": ast.literal_eval(key)['external_id'],
+            "_links.sequencing_center": (
+                ast.literal_eval(key)['sequencing_center_id']
+            ),
+        }
+        rows.append(row_dict)
+    return pd.DataFrame(rows)
+
+
+@pytest.fixture
+def mock_get_entities(mocker, study_generator):
+    """
+    Mock for creator.ingest_runs.utils.get_entities
+    """
+    def fake_get_entities(*args, **kwargs):
+        PAYLOAD_MAP = {
+            "genomic-files": map_genomic_file,
+            "sequencing-experiment-genomic-files": map_seq_exp_gen_file,
+            "sequencing-experiments": map_seq_exp,
+        }
+        entity_type = args[1]
+        return PAYLOAD_MAP[entity_type](study_generator)
+
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.utils.get_entities",
+        side_effect=fake_get_entities
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_load_entities(tmpdir, mocker, study_generator):
+    def _dry_load_entities(*args, **kwargs):
+        """
+        Inserts KF IDs into output of _load_entities
+        """
+        from kf_lib_data_ingest.app.settings.base import TARGET_API_CONFIG
+        entity_type = args[0]
+        loader = LoadStage(
+            TARGET_API_CONFIG,
+            DATASERVICE_URL,
+            [entity_type],
+            study_generator.study_id,
+            cache_dir=os.path.join(tmpdir, "temp"),
+            dry_run=True,
+        )
+        loader.run({entity_type: args[1]})
+        cache = loader.uid_cache
+
+        return cache
+
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "_load_entities",
+        side_effect=_dry_load_entities
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_load_bs_gf_links(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "load_specimen_harmonized_gf_links"
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_load_se_gf_links(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "load_seq_exp_harmonized_genomic_files"
+    )
+    return mock
+
+
+@pytest.fixture
+def mock_validate(mocker):
+    mock = mocker.patch(
+        "creator.ingest_runs.genomic_data_loader.GenomicDataLoader."
+        "_validate_load"
+    )
+    return mock
+
+
+def test_load_harmonized_genomic_files(
```

Comment:
Since you're only testing loading of the genomic files here, you probably don't need all of the mocks. You could just do with the mocks that would be called in `load_harmonized_genomic_files`. In your test here you can change `loader.ingest_gwo(manifest_df)` to `df = loader.load_harmonized_genomic_files(manifest_df)`